### DISCUSSION TITLE
Is there any ways to increase the speed?

### DISCUSSION THREAD

**Discussion by Raizudeen-S** [SWivid/F5-TTS#202] [Q&A]
Hey for me it took more than 10 sec to inference to audio for a text sentence.  I am try to use it with an LLM but the inference takes much longer. Is there any way or better method then provided to speed up the inference speed of the model?

**Comment by lpscr** [comment]
@Raizudeen-S 

use api and you can change nfe_step to 32,24,16 see what working best for you

```
from api import F5TTS

f5tts=F5TTS()

wav,sr,spect=f5tts.infer(
    ref_file="tests/ref_audio/test_en_1_ref_short.wav",
    ref_text="Some call me nature, others call me mother nature.",
    gen_text="""I don't really care what you call me.""",
    nfe_step=16,
    file_wave="test.wav",
)
```

**Reply by Raizudeen-S** [reply]
Thanks for your Response. Is there any other method I can try to to improve the speed. GPU memory usage it not a problem for me.
